webpack.config.babel.js

```diff
@@ -208,17 +209,17 @@ const configDist = {
     }),
     ... PRODUCTION_BUILD ?
       [
-        new webpack.optimize.UglifyJsPlugin({
-          beautify: false,
+        new UglifyJSPlugin({
           sourceMap: true,
-          compress: {
```

Comment:
I removed all config that was using the default setting. The format has also slightly changed, but the options are the same